runtime/src/main/java/org/corfudb/runtime/Backup.java

```diff
@@ -265,11 +290,36 @@ private void addToTarFile(File tableFile,TarArchiveOutputStream tarOutput) throw
                 .map(tableName -> CorfuRuntime
                         .getStreamID(TableRegistry.getFullyQualifiedTableName(tableName)))
                 .collect(Collectors.toList());
-        log.info("{} tables need to be backed up: {}", tables.size(), tables);
+        log.info("{} tables need to be backed up.", tables.size());
```

Comment:
prev version is a bit better in terms of logs parsing

Reply:
`tables` is usually a very large set and considering there is already logging for each table, it could be cleaner to not log them again here.
`log.info("{} entries (elapsed time {}ms) saved to temp file {}", numberOfEntries, elapsedTime, filePath);`
(temp file has both table UUID and table name info)